src/plugins/discover/public/application/main/components/layout/discover_documents.tsx

```diff
@@ -93,7 +93,7 @@ function DiscoverDocumentsComponent({
       const grid = { ...(state.grid || {}) };
       const newColumns = { ...(grid.columns || {}) };
       newColumns[colSettings.columnId] = {
-        width: colSettings.width,
+        width: Math.round(colSettings.width),
```

Comment:
thx for fixing 🙏 , I've one ask for this, could you add a unit test for this, maybe extracting this function so it can be done easily?